tests/utils_tests/testing_tests/test_constant_stub_link.py

```diff
@@ -0,0 +1,78 @@
+import numpy as np
+import unittest
+
+import chainer
+from chainer import testing
+from chainer.testing import attr
+
+from chainercv.utils import ConstantStubLink
+
+
+@testing.parameterize(
+    {'outputs': {'shape': (3, 4), 'dtype': np.float32}},
+    {'outputs': ({'shape': (3, 4), 'dtype': np.float32},)},
+    {'outputs': (
+        {'shape': (3, 4), 'dtype': np.float32},
+        {'shape': (3,), 'dtype': np.int32})},
+)
+class TestConstantStubLink(unittest.TestCase):
+
+    def setUp(self):
+        if isinstance(self.outputs, tuple):
+            self.outputs = tuple(np.empty(**output) for output in self.outputs)
```

Comment:
`np.random.uniform` is better because it is guaranteed that the output is not all 0 (which is a trivial test case).